build_run_docker.sh

```diff
@@ -433,6 +433,7 @@ service-template() {
     %s:
       image: %s
       container_name: %s
+      user: $(id -u):$(id -g)
```

Comment:
@svenssonaxel, this is kinda key. I wasn't aware earlier that this was possible.